src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -2930,7 +2930,8 @@ void CalcIndirectRDDEvapCoolerOutletTemp(EnergyPlusData &state,
             CapFlowSys = thisEvapCond.InletMassFlowRate * CpAirSys;
             QHXRate = CapFlowSys * (thisEvapCond.InletTemp - OutletTemp);
             SecOutletEnthalpy = thisEvapCond.SecInletEnthalpy + QHXRate / AirMassFlowSec;
-            SecOutletAirHumRat = Psychrometrics::PsyWFnTdbH(state, EDBTSec, SecOutletEnthalpy); // assumes constant temperature moisture addition
+            SecOutletAirHumRat = Psychrometrics::PsyWFnTdbH(state, EDBTSec,
+                                                            SecOutletEnthalpy); // assumes constant temperature moisture addition
```

Comment:
Is this just a comment-only change?  I don't see anything different besides it split into two lines.

Reply:
I think it just reformatted the code to two lines. Sometimes the clang-format tries to align the starting point of the comments. So even if this particular line is not changed, sometimes the formatting wants it differently based on changes of other lines. That might be what's happening here.